app/controller/command/commands/team.py

```diff
@@ -442,7 +458,17 @@ def remove_helper(self, param_list, user_id) -> ResponseTuple:
             self.gh.remove_team_member(user.github_username,
                                        team.github_team_id)
             self.facade.store(team)
-            msg = "Removed User from " + param_list['team_name']
+
+            msg = "Removed User from " + command_team
+
+            # If this team is a team with special permissions, demote the user
+            if command_team == self.config.github_team_admin\
+                    or command_team == self.config.github_team_leads:
+                logging.info(f"Demoting {command_user} to member")
+                user.permissions_level = Permissions.member
+                self.facade.store(user)
+                msg += " and demoted user"
+
```

Comment:
User may be both admin and team lead - if they are removed from admin rank, they should go to team lead. If they are removed from team leads, they should remain an admin.

Reply:
Good catch, thanks! ea1f30c721f5010eb496b92ff72c5209a900cd72